experimental/streaming/exporter/observer/observer.go

```diff
@@ -85,73 +82,42 @@ const (
 	SET_NAME
 )
 
-var (
-	observerMu sync.Mutex
-	observers  atomic.Value
-
-	sequenceNum uint64
-)
-
-func NextEventID() EventID {
-	return EventID(atomic.AddUint64(&sequenceNum, 1))
+type Exporter struct {
```

Comment:
Should that type go to the `experimental/streaming/exporter` package then?

Reply:
Done.